example/gluon/style_transfer/utils.py

```diff
@@ -0,0 +1,119 @@
+import os
+from PIL import Image
+
+import numpy as np
+import mxnet as mx
+import mxnet.ndarray as F
+
+def tensor_load_rgbimage(filename, ctx, size=None, scale=None, keep_asp=False):
```

Comment:
@zhreshold what would you recommend to use for these image transformation methods?

Reply:
@zhreshold is there an option to resize the image along the short edge and keep aspect ratio? Thanks!